go.sum

```diff
@@ -1,5 +1,7 @@
-github.com/aws/aws-sdk-go v1.49.13 h1:f4mGztsgnx2dR9r8FQYa9YW/RsKb+N7bgef4UGrOW1Y=
-github.com/aws/aws-sdk-go v1.49.13/go.mod h1:LF8svs817+Nz+DmiMQKTO3ubZ/6IaTpq3TjupRn3Eqk=
+github.com/RueLaLa/session-manager-plugin v1.3.1 h1:O85kF/AZcwcjJeEbg9m0YIuDJqOmQA0AN1oleaZ68eY=
+github.com/RueLaLa/session-manager-plugin v1.3.1/go.mod h1:kpIkvJv/BH88I5YN4Qh9Geq9FbCa1D9WtiRZsF37PuU=
```

Comment:
Note: the aws version wasnt present, since it was local previously:
```
A replace directive replaces the contents of a specific version of a module, or all versions of a module, with contents found elsewhere. The replacement may be specified with either another module path and version, or a platform-specific file path.
```